am/routes.py

```diff
@@ -158,9 +158,9 @@ def on_post(self, req: falcon.Request, resp: falcon.Response, store_id: str, pro
         try:
             meta = self._controller.get_asset_meta(store_name=store_id, project_name=project_id, asset_name=asset_id)
             if meta.uploaded:
-                raise falcon.HTTPConflict(title="Asset exists", description="This asset already has a file. If you wish to change this file, use update")
+                raise falcon.HTTPConflict(title="Asset exists", description="This asset already has a file. If you wish to change this file, check the 'Update content' checkbox.")
```

Comment:
This is a backend message, so there is no concept of a checkbox here

Reply:
Good point.

'Use update' is a good message for the API to return, as the API has a separate update method. However, there is no  'update' action distinct from 'upload' in the UI. 

One solution is for the UI to display error messages that are different to those returned by the backend.

An alternative is for the UI to prevent the user from initiating an upload action if the asset already has a file unless they check the confirmation box; this should prevent this error condition from ever being reached. However, there might be a problem if an upload has already started from a different browser window.